spec/lib/csv_importer_spec.rb

```diff
@@ -36,4 +36,25 @@
       end
     end
   end
+
+  describe "#process" do
+    let(:processed_file) { create(:processed_file) }
+    let(:category_name) { "Measurement" }
+    let(:file) { File.read(Rails.root.join("spec/fixtures/basic_custom_measurement.csv")) }
+    context "allows the upload of custom tank measurments (without notes)" do
```

Comment:
Probably not important? spelling on `Measurement`.
Relevant if we think code editors will pick up on it and autofill to misspellings.